unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -2597,6 +2598,79 @@ bool includeInvisible
             actionFinished(true);
         }
 
+        public void Get2DSemanticHulls(List<string> objectTypes = null) {
```

Comment:
1. `objectTypes` actually isn't required, if it is `null` then it is assumed that all object types should be used. That said, I've changed the api to
```python
        public void Get2DSemanticHulls(
            List<string> objectIds = null,
            List<string> objectTypes = null
        ) {
```
so that a user can specify ids as well.

2. The main reason to have `objectTypes` is that this function is targeting semantic information which is inherently object-type-centric. It's "easy" to handle this all on the python side but is also a bit annoying in practice since you have to do it every time you call the function (or at least I would).

3. This overload is now in.

I've also added a comment:
```python
        /*
        For each objectId, create a convex hull of the object from a top-down view.
        The convex hull will be represented as a list of (x, z) world coordinates
        such that the boundary formed by these coordinates forms the convex hull
        of these points (smallest convex region enclosing the object's points).

        If the objectIds (or objectTypes) parameter is non-null, then only objects with
        those ids (or types) will be returned.
        
        ONLY ONE OF objectIds OR objectTypes IS ALLOWED TO BE NON-NULL.

        Returns a dictionary mapping object ids to a list of (x,z) coordinates corresponding
        to the convex hull of the corresponding object.
        */
```